helm/chaos-mesh/templates/_helpers.tpl

```diff
@@ -1,42 +1,61 @@
 {{/* vim: set filetype=mustache: */}}
+
+{{/*
+Handle env variables.
+
+TODO: in the future, we would like to use the k8s-like format for defining environment variables.
+So the `envFollowKubernetesPattern` will become to `env`.
+And the original way of writing env will be removed.
+Ref: https://github.com/chaos-mesh/chaos-mesh/pull/2955.
+*/}}
+{{- define "chaos-mesh.helpers.list-env-vars" -}}
```

Comment:
According to [helm chart best practice](https://helm.sh/docs/chart_best_practices/values/#naming-conventions), maybe we should use `listEnvVars` instead of `list-env-vars`.

Reply:
Now fixed!